### PR TITLE
an-error-in-the-console-fixed

### PR DESCRIPTION
**Pull request by Rizzwaan**
jsx syntax error was fixed

### CODE REVIEW

File: src/components/Footer/contributors.js [hunk 1]
```diff
@@ -1,56 +1,69 @@
 import React from 'react';
-import './contributors.css'
+import './contributors.css';
 
-const Contrib = ({contributors}) => {
+const Contrib = ({ contributors }) => (
+  <div>
+    <div className="cFooterlink" data-toggle="modal" data-target="#contribModal">
+      <li>& {contributors.length} Contributors</li>
```
Comment (aneagoie) [inline src/components/Footer/contributors.js]: Could we wrap this in a template string so it is clear what we are displaying here?

**Comment by aneagoie** [comment]
Good job! Just one fix needed, which I have added a comment for. Let me know when it is ready to be merged :)

File: src/components/Footer/contributors.js (hunk 1 above)
Comment (Rizzwaan) [reply src/components/Footer/contributors.js]: I can wrap it in a return statement also

**Comment by MattCSmith** [comment]
@Rizzwaan Nice, could you kindly address the conflict error :)

**Pull request merged**